src/Raven.Client/Documents/Session/Events.cs

```diff
@@ -1,12 +1,11 @@
 ﻿using System;
 using System.Collections.Generic;
-using Microsoft.Extensions.Primitives;
 
 namespace Raven.Client.Documents.Session
 {
     public class BeforeStoreEventArgs : EventArgs
     {
-        private IDictionary<string, StringValues> _documentMetadata;
+        private IDictionary<string, object> _documentMetadata;
```

Comment:
I changed that.
The reason that I introduced IMetadataDictionary after changing this... but now I fixed this. Thanks.